flyway-core/src/main/java/org/flywaydb/core/Flyway.java

```diff
@@ -155,6 +155,30 @@
     private String sqlMigrationSuffix = ".sql";
 
     /**
+     * The file name prefix for shell migrations. (default: V)
+     * <p/>
+     * <p>Shell migrations have the following file name structure: prefixVERSIONseparatorDESCRIPTIONsuffix ,
+     * which using the defaults translates to V1_1__My_description.shell</p>
+     */
+    private String shellMigrationPrefix = "V";
+
+    /**
+     * The file name separator for sql migrations. (default: __)
+     * <p/>
+     * <p>Shell migrations have the following file name structure: prefixVERSIONseparatorDESCRIPTIONsuffix ,
+     * which using the defaults translates to V1_1__My_description.shell</p>
+     */
+    private String shellMigrationSeparator = "__";
+
+    /**
+     * The file name suffix for shell migrations. (default: .sql)
```

Comment:
Fixed in https://github.com/rollbar/flyway/commit/6b5c6f22b51e8b93c63bbcd25d5ad39557d6559a